### DISCUSSION TITLE
x-init does not work properly on the blade file

### DISCUSSION THREAD

**Discussion by NguyenTheSon** [alpinejs/alpine#1392] [7. Bugs (archived)]
I wrote a blade file. I then used this file many times using @include the laravel. x-data only gets the last value.
Meanwhile I console log in $(document).ready. it got full the data
Please support me
![Capture121](https://user-images.githubusercontent.com/12813174/113330083-90ba7800-9348-11eb-85fc-4fac5971063c.png)

**Comment by KevinBatdorf** [comment]
Can you share some code? What's in your `x-init`?

**Comment by NguyenTheSon** [comment]
It's some my code
![image](https://user-images.githubusercontent.com/12813174/113372726-8376ab00-9393-11eb-8c3f-a9a5affd74a8.png)
![image](https://user-images.githubusercontent.com/12813174/113373030-27605680-9394-11eb-9e28-77624a31b69e.png)

**Comment by SimoTod** [comment]
Your code is slightly different. in document ready you are not doing [...@json($slides)] so you are outputting different things.

**Comment by NguyenTheSon** [comment]
Okay. I understand. Thanks you!
